cattle/plugins/docker/compute.py

```diff
@@ -296,8 +298,9 @@ def to_upper_case(key):
 
         container = self.get_container_by_name(name)
         if container is None:
-            log.info('Creating docker container [%s] from config %s', name,
-                     create_config)
+            log.info('Creating docker container [%s]'
+                     'from config %s using image (%s)',
+                     name, create_config, image_tag)
```

Comment:
We have a *loose* convention of putting variable inside of square brackets in log statements. We don't push it hard or strictly follow it all the time, but I wanted to call attention to it here because I see that you have three different paradigms in this log statement: square brackets, no brackets, and parentheses. Perhaps just consistently go with square brackets.